### DISCUSSION TITLE
对apollo部署架构的疑惑

### DISCUSSION THREAD

**Discussion by zJiaJun** [apolloconfig/apollo#4281] [Q&A]
portal 访问各环境的meta域名，来获取admin和config服务的ip:port 来访问
这个slb 域名背后会配置 meta + config 服务的ip吧，这里的ip应该是固定的吧，即来都固定了，为什么不直接连接config服务（meta 和 config是同一个进程），而是服务发现config服务，是我理解不对吗？
还是因为有admin服务的存在，假设只有一个config服务，有和meta同进程，就没必要服务发现逻辑吧

**Comment by nobodyiam** [comment]
see #1748
